projects/packages/jetpack-mu-wpcom/changelog/verbum-add-gravatar-comments

```diff
@@ -0,0 +1,3 @@
+Significance: patch
```

Comment:
Not sure what's the usual decision on this codebase, but this seem more like a new feature than a patch.

Reply:
Ah yep. It wouldn't let me create a PR without one of these files so I just copy/pasted one, and then forgot about it.